webhook.rst

```diff
@@ -0,0 +1,280 @@
+Webhook
+=======
+
+.. versionadded:: 6.3
+
+    The Webhook component was introduced in Symfony 6.3 and is marked
+    as experimental.
+
+The Webhook component aims at easing the setup of callback functions
+through HTTP in your application.
+
+Webhooks allow third-party services to communicate with your application
+without you having to make a request to the third-party application.
+By providing a webhook URL to the remote service, it will be able to
+send a request to this URL when a predefined event occurs.
+Unlike an API, it is the third-party service that defines the payload that
+will be sent to your own endpoint. It is therefore up to you to adapt to
+the service.
+
+A recurrent use case is the return of the mail provider.
```

Comment:
As it may not be that recurrent, what about just add instead: As an example use case, email providers can send webhook…